src/components/Bookmark/Bookmark.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+import {useState} from 'react';
+import BookmarkIcon from '../BookmarkIcon/BookmarkIcon';
+import NoBookmarkIcon from '../NoBookmarkIcon/NoBookmarkIcon';
+
+export default function Bookmark({onChangeBookmark, bookmarked, id}) {
+  const [bookmark, setBookmark] = useState(bookmarked);
```

Comment:
You dont't need this state at this point. You can use the bookmarked prop of your exercise instead.